lib/tree_sitter/node.rb

```diff
@@ -160,34 +160,18 @@ def to_a
     # uses         named_child                   | field_name_for_child
     #              child_by_field_name           |   via each_node
     #              ------------------------------+----------------------
-    # @param all [Boolean] If `true`, return an array of nodes for all the
-    # demanded keys, putting `nil` for missing ones.  If `false`, return the
-    # same array after calling `compact`. Defaults to `false`.
     #
     # See {#fetch_all}.
-    def fetch(*keys, all: false, **_kwargs)
-      key_set = keys.to_set(&:to_s)
+    def fetch(*keys, **_kwargs)
```

Comment:
I'm sure to understand why we accept extraneous kwargs.  I would get rid of them.  Or leave them unnamed (`**`).